application/actions/actions_test.go

```diff
@@ -95,3 +99,13 @@ func createFixture(as *ActionSuite, f interface{}) {
 		as.T().FailNow()
 	}
 }
+
+func (as *ActionSuite) verifyResponseData(wantData []string, body string) {
+	var b bytes.Buffer
+	as.NoError(json.Indent(&b, []byte(body), "", "    "))
+	for _, w := range wantData {
+		if !strings.Contains(body, w) {
+			as.Fail(fmt.Sprintf("response data is not correct\nwanted: %s\nin body:\n%s\n", w, b.String()))
```

Comment:
This error message seems like it would make debugging a test easier, and messages like this are something we may want to start adding to other unit tests using the Contains/string-matching approach.